services/notebook/notebook.go

```diff
@@ -20,8 +20,9 @@ import (
 )
 
 type NotebookManager struct {
-	config_obj *config_proto.Config
-	Store      NotebookStore
+	config_obj   *config_proto.Config
+	Store        NotebookStore
+	VqlProcessor *VqlProcessor
```

Comment:
There is probably no value in separating this into a processor since we dont really need to switch implementations.